src/tutorial/hello-world/README.md

```diff
@@ -20,7 +20,32 @@ Each part will look as follows for the string `"Hello, world!"`:
 0x40            48656c6c6f2c20776f726c642100000000000000000000000000000000000000 // Value "Hello, world!"
 ```
 
-Encoding dynamic values takes alot of work!! In order to return `"Hello, world!"` we must return 96 bytes!.
+Encoding dynamic values takes alot of work!! In order to return `"Hello, world!"` we must return 96 bytes!
+
+You might think that we can populate the memory in sequential order starting with the location (0x00), then length (0x20), and then the value (0x40), however, this would result in the value "Hello, world!" being left padded, so we would get: 
```

Comment:
```suggestion
You might think that we can populate the memory in sequential order starting with the location (0x00), then length (0x20), and then the value (0x40). However, this would result in the value "Hello, world!" being left padded. We would get: 
```